src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -39,8 +43,34 @@
      * Андрей - 600
      * София - 900</p>
      */
+    private static final DateTimeFormatter DATE_FORMAT = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
+    private boolean isDateInRange(String date, LocalDate startDate, LocalDate endDate) {
```

Comment:
private methods should be located after public, so let's move it after our getSalaryInfo method